views/gnome/email/papers/paper_submitted.hbs

```diff
@@ -2,4 +2,10 @@ Your paper has been submitted
 
 Hi,
 
-Thank you for submitting your paper with title {{paper.title}} for the {{paper.track}} track.
+Thank you for submitting your talk titled "{{paper.title}}" for the {{paper.track}} track.
```

Comment:
Probably not tracks, but I think it makes sense to offer shorter and longer talks, but I haven't figured out how to best present that,